packages/investor-foxy/src/api/api.ts

```diff
@@ -462,33 +554,26 @@ export class FoxyApi {
     if (!amountDesired.gt(0)) throw new Error('Must send valid amount')
     if (!wallet) throw new Error('Missing inputs')
 
-    let estimatedGasBN: BigNumber
+    let estimatedFees: FeeDataEstimate<KnownChainIds.EthereumMainnet>
     try {
-      estimatedGasBN = await this.estimateDepositGas(input)
-    } catch (err) {
-      throw new Error(`Estimate Gas Error: ${err}`)
+      estimatedFees = await this.estimateDepositFees(input)
```

Comment:
seems like an unnecessary try/catch imo. just const and let the function throw if fail

Reply:
a handful of these, not new, but just noticed

Reply:
9726c2fad363f8baa4daa943f028e74780b8bdd7